client/src/components/maps/MyMap.js

```diff
@@ -1,12 +1,55 @@
-import { GoogleMap } from "react-google-maps";
+import React from "react";
```

Comment:
I don't think you need to import React anymore, but depends on the version you're using.